keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/operators/multiple/ProcessModelVertexTest.java

```diff
@@ -30,17 +30,14 @@
 public class ProcessModelVertexTest {
 
     /*
-    The model we are mimicking here is a python script, model.py.
```

Comment:
I wouldn't delete this as the information about the return types of the model is useful to show people how to link up the `result output vertices` to this model and it explains my expected results in the tests. I would just add to this by saying "for performance reasons we do not actually run the python script and instead just mock its behaviour"